src/main/java/domain/LottoResult.java

```diff
@@ -0,0 +1,47 @@
+package domain;
+
+import dto.LottoResultDto;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoResult {
+    private List<Integer> result;
```

Comment:
자바 컨벤션에 따르면 한 줄 띄어쓰기가 필요합니다
```suggestion
public class LottoResult {

    private List<Integer> result;
```

Reply:
현재 LottoResult는 현재 책임이 하나도 없는데 객체로 관리하는 이유는 무엇인가요?